doc/usecases/record_elektra/UC_interactive_configuration.md

```diff
@@ -0,0 +1,25 @@
+# Use Case: Interactively creating system config
+
+## Summary
+
+- **Title:** Interactively creating system config
+- **Scope:** Configuration
+- **Level:** User Goal
+- **Actors:** User (usually sysadmin)
+- **Brief:** In order to interactively create a workable system configuration, sometimes multiple iterations are required. 
+It should be possible to modify the configuration on a single host, and then apply the changes onto all other hosts.
```

Comment:
Why limit to modifications on a single host? As long the changes can somehow be merged...

```suggestion
It should be possible to modify the configuration directly on hosts, and then apply the changes onto all other hosts.
```

Reply:
The problem is, if you change the configuration on multiple systems at the same time, we'd have to manually merge the playbooks - as in the user would have to copy all the playbooks from the machines and let a tool merge them. 

If we limit this to a single host at a time, the ansible export plugin could do this automatically at this host.